pkg/workflows/upgrade.go

```diff
@@ -652,6 +662,11 @@ func (s *deleteBootstrapClusterTask) Run(ctx context.Context, commandContext *ta
 			// We don't necessarily need to return with an error here and abort
 			logger.Info(fmt.Sprintf("%v", err))
 		}
+
+		capiObjectFile := filepath.Join(commandContext.BootstrapCluster.Name, logger.GetManagementClusterStatePath())
+		if err := os.RemoveAll(capiObjectFile); err != nil {
+			logger.Info(fmt.Sprintf("management cluster CAPI backup file not found: %v", err))
```

Comment:
```suggestion
			logger.Info(fmt.Sprintf("deleting management cluster CAPI backup: %v", err))
```

Reply:
Since this is not throwing any error, does this look okay?
`logger.Info(fmt.Sprintf("management objects backup folder not found: %v", err))`

Reply:
How about `Warning: Unable to find management objects backup folder due to: %v`?